### DISCUSSION TITLE
Kernel crash in docker

### DISCUSSION THREAD

**Discussion by EngSubsurfaceModeling** [gempy-project/gempy#662] [General]
I followed instruction and pulled docker image. start jupyter, and tried the "get started".

![image](https://user-images.githubusercontent.com/98186456/152626564-fa8a1917-2d8b-49e6-89a0-17fdffea690a.png)
![image](https://user-images.githubusercontent.com/98186456/152626572-feedfa8b-98c4-4dde-add5-da69957505d0.png)

when run "p3d = gp.plot_3d(geo_model)", kernel died.  any idea?

**Comment by Japhiolite** [comment]
Hi @EngSubsurfaceModeling 

do you have an error message, e.g. from the Terminal? Crashing Kernel can have a multitude of different reasons, most common with `plot_3d` are some problems with qt5 and pyqt5.

Edit: It might also be good to move this to the Issues, instead of having it here in Discussions.

Best,
Jan
